.github/fabricbot.json

```diff
@@ -1059,6 +1060,25 @@
               ""
             ]
           },
+          {
+            "label": "ML-Local Endpoints",
+            "pathFilter": [
+              "sdk/ml/azure-ai-ml/azure/ai/ml/_local_endpoints/"
```

Comment:
Looks like this is a private path?

Reply:
It was added as part of a CODEOWNERS change in the linked PRs.    Should this not exist in either place?

Reply:
Double checking with ML team.

Reply:
We decided to discuss the issue later. :)